server/src/models/swing.rs

```diff
@@ -179,12 +179,22 @@ impl Swing {
         }
     }
 
+    /// Return the number of values the model currently represented.
+    pub fn get_length(&self) -> usize {
+	self.length
```

Comment:
See my response to this [comment](https://github.com/ModelarData/ModelarDB-RS/pull/23#discussion_r944133684).

Reply:
Somehow the indention for `self.length` was done using a tab instead of spaces so it looked fine to me with special characters hidden. I have replaced the tab with spaces.